src/SignalingClient.ts

```diff
@@ -158,6 +159,36 @@ export class SignalingClient extends EventEmitter {
         }
     }
 
+    public reconnect(maxAttempt = 3): void {
+        this.reconnection = true;
+        if (this.readyState === ReadyState.CLOSED) {
+            this.attemptReconnect(maxAttempt);
+        } else {
+            console.log('Invalid state to invoke reconnect from');
+        }
+    }
+
+    private attemptReconnect(maxAttempts: number, attempt = 0): void {
+        if (this.readyState === ReadyState.CLOSED) {
+            this.open();
+            // If reconnect fails
+            setTimeout(() => {
+                if (this.readyState !== ReadyState.OPEN) {
+                    if (attempt < maxAttempts) {
+                        console.log("Failed reconnect attempt ", attempt);
```

Comment:
console.log("Failed reconnect attempt ", attempt);
```suggestion
                        console.log("Failed reconnect attempt", attempt);
```
Space not needed, this will result in double space